### DISCUSSION TITLE
Memindahkan Radio Buttons & Box ke Tengah

### DISCUSSION THREAD

**Discussion by namiraprita** [teamalgoritma/community#256] [Q&A] [answered]
Hi Algoritma Team, 

Saya ingin mencoba untuk memindahkan "All You Need to Know About US Dat Science Salary" dan radio buttons ke tengah dan sejajar. Bagaimana kah caranya?

![Screen Shot 2022-08-05 at 12 51 59](https://user-images.githubusercontent.com/107253301/183010479-5b92bd64-a235-406e-825c-2f33abc9dba4.png)

Berikut code UInya:
![Screen Shot 2022-08-05 at 12 52 36](https://user-images.githubusercontent.com/107253301/183010541-c72f5547-d247-4bdd-a215-82b16c844661.png)

Terima kasih!

**Comment by dwiknrd** [comment, marked as answer]
Hai [namiraprita](https://github.com/namiraprita),
Untuk meletakkan text html di posisi tengah, anda dapat menambahkan parameter `align = "center"` pada fungsi box yang sudah dilampirkan diatas.

Semoga membantu :)
